### DISCUSSION TITLE
Review Thread: Posteo

### DISCUSSION THREAD

**Discussion by lissy93** [Lissy93/email-comparison#23] [Reviews]
<h1 align="center"><sup>User Reviews</sup><br>Posteo</h1>

This is a place for you to share your thoughts and experiences using this email provider. The aim is to help other users make an informed discussion when choosing a private and/or secure email service.

Don't forget to upvote the reviews you've found helpful!

<p align="center">
<sup>Click the card below, to view info about this email service</sup>
<a href="https://email-comparison.as93.net/email/posteo">
<img width="800" src="https://api.apiflash.com/v1/urltoimage?access_key=97d02977152a442a82c708d3efe53e41&url=https%3A%2F%2Femail-comparison.as93.net%2Femail%2Fposteo%2Fsummary&format=png&width=1500&ttl=2592000&response_type=image&element=body" />
</a>
</p>

---

<blockquote>
<h3>Important</h3>
<sup>
    If you are associated with a given service or it's direct competitors in an form, you MUST disclose your affiliation<br>
    Please note that your review will be visible on the website, along with your GitHub name, profile age and avatar<br>
    No review will ever be hidden / deleted, unless it is obviously spam or actively offensive<br>
    By posting your review, you agree to follow the Contributor Code of Conduct<br>
    Basic markdown formatting is supported (links, scaled images, bold, italics)<br>
</sup>
</blockquote>

**Comment by NA0341** [comment]
**09/10 A Great E-Mail Provider!**

> I live in Germany and my first E-Mail - Account was a GMX Account from 1&1 Mail & Media GmbH.
> It worked fine, but the "Free" Plan is full of Spam & Ads from the Provider. I switched to the Paid Version which was rather expensive and I still had a Login Page full of Ads.
> So, to put it mildly, Not the Greatest User Experience.

> After Some Time I was Truly fed up and searched for an Alternative. I checked Mailbox.org (which has the "Mobile Office" Approach) and Tuta (Tutanota at the time) among others. Tutanota appeared pretty weird to me with a Website full of Ninja's and a message that was like "We Make E-Mail Great again" or whatever. Not what I want to entrust my Personal Address for the many Years to Come to.

> I checked out Posteo ~ and found out: **It's Great!**

I'm a Minimalist, and Posteo has that Approach.

+ Great Web-UI (with drag & drop, compact view, dynamic themes, ..), Features-on-Demand, Rock-Solid ~ Stable ~ Reliable, No-Nonsense, Sensible Defaults
+ …and Lot's of Lifechanging Extras such as a Collector Service that Pulls Mails from Other Accounts, Migration Service which transfers the entire Mail & Folder Structure; CalDAV, CardDAV - Support, etc. pp.
+ …and Neat Things such as the Possibility to gift Credit.

There are Some Minor Drawbacks such as the Lack for Contact Groups (which is a Feature most Platforms miss ~ even though it's really a Necessity at some point! Especially when syncing with your Phone using i.e. Davx5) ~ but Besides That I can't think of another thing I'm missing right now.

I started using Posteo in mid 2020 and will most certainly for Many Years to Come.
The best indicator: I've never really thought about the Provider itself after Switching.
And that's a Good Sign. (Like with all good decisions :)

Lastly, here's how the Web-UI looks like (with My Settings):
![image](https://github.com/user-attachments/assets/59fa842e-6a13-404f-8dac-06c1a84bb4be)

Psst. Regarding Spam: Posteo has integrated Spam-Protection that works like a Firewall. So Spam won't even reach your inbox. There is an option to enable the Spam Folder for those who really want to.
Additionally, there's Settings for a TLS Sending & Receiving Guarantee which will make Posteo Servers Refuse Any E-Mail that Can't be Sent Over a Secure Connection (outgoing and or incoming).
This further reduces the change of Spam reaching your Inbox.

In the Five Years I've been using Posteo there wasn't a single of the Thousands of Mails which was Falsely Flagged as Spam and Rejected.
